ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -16,9 +17,15 @@ import {
 } from '../../../../shared/constants/metametrics';
 
 import { I18nContext } from '../../../contexts/i18n';
+import { MULTICHAIN_ACCOUNT_ADDRESS_LIST_PAGE_ROUTE } from '../../../helpers/constants/routes';
+import {
+  AddressListQueryParams,
+  AddressListSource,
+} from '../../../pages/multichain-accounts/multichain-account-address-list-page';
```

Comment:
Importing react router hook and address routing helpers for navigating to the receive page via the Add funds modal